api/pom.xml

```diff
@@ -21,7 +21,7 @@
     <parent>
         <artifactId>grizzly-npn</artifactId>
         <groupId>org.glassfish.grizzly</groupId>
-        <version>2.0.1-SNAPSHOT</version>
+        <version>2.0.2-SNAPSHOT</version>
```

Comment:
This isn't aligned with our usual versioning strategy.
Should be `2.0.0.payara-p1` (followed by `2.0.0.payara-p2-SNAPSHOT`) - we shouldn't be touching the "actual" version so as to not collide with upstream if they do a 2.0.1 release.

Reply:
I've deleted the 2.0.1 version uploaded to payara-artifacts, as it's incorrect.